lib/samples/create_mobile_geodatabase/create_mobile_geodatabase_sample.dart

```diff
@@ -0,0 +1,353 @@
+    if (_featureTable == null) {
+      return;
+    }
+
+    final attributes = {
+      'collection_timestamp': DateTime.now(),
+    };
+    final newFeature = _featureTable!.createFeature(
+      attributes: attributes,
+      geometry: point,
+    );
+    await _featureTable!.addFeature(newFeature);
+    setState(() => _featureCount = _featureTable!.numberOfFeatures);
+  }
+
+  // Display the attribute table in a dialog.
+  void _displayTable() async {
+    final queryResult =
+        await _featureTable?.queryFeatures(parameters: QueryParameters());
+
+    List<DataRow> dataRow = [];
+    for (final f in queryResult!.features()) {
+      dataRow.add(
+        DataRow(
+          cells: [
+            DataCell(
+              Text(
+                f.attributes['oid'].toString(),
+              ),
+            ),
+            DataCell(
+              Text(
+                f.attributes['collection_timestamp'].toString(),
+              ),
+            ),
+          ],
+        ),
+      );
+    }
+    if (mounted) {
+      showDialog(
+        context: context,
+        barrierColor: Colors.transparent,
+        builder: (context) {
+          return SimpleDialog(
+            shape: const RoundedRectangleBorder(
+              borderRadius: BorderRadius.zero,
+            ),
+            children: [
+              Padding(
+                padding: const EdgeInsets.fromLTRB(15, 5, 15, 10),
+                child: SingleChildScrollView(
+                  scrollDirection: Axis.vertical,
+                  child: DataTable(
+                    border: TableBorder.all(),
+                    columns: const [
+                      DataColumn(
+                        label: Text('OID'),
+                      ),
+                      DataColumn(
+                        label: Text('Collection Timestamp'),
+                      ),
+                    ],
+                    rows: dataRow,
+                  ),
+                ),
+              ),
+              const Padding(
+                padding: EdgeInsets.fromLTRB(15, 0, 15, 10),
+                child: Text(
+                  'Attribute table loaded from the mobile geodatabase '
+                  'file. File can be loaded on ArcGIS Pro or ArcGIS Runtime.',
+                  style: TextStyle(
+                    fontSize: 12.0,
+                  ),
+                ),
+              ),
+            ],
+          );
+        },
+      );
+    }
+  }
+
+  // Call platform share sheet and share the mobile geodatabase file URI.
+  void _shareGeodatabaseUri() async {
+    _geodatabase?.close();
+
+    final shareResult = await Share.share(
+      subject: 'Sharing the geodatabase',
+      _geodatabase!.fileUri.path,
+    );
+
+    if (shareResult.status == ShareResultStatus.success) {
+      _showDialog('Success', 'The geodatabase was shared successfully.');
+    } else {
+      _showDialog('Error', 'The geodatabase could not be shared.');
+    }
```

Comment:
Now that `_geodatabase` has been closed, what happens after the share workflow? Should the geodatabase be re-opened? Or should a new empty geodatabase be created?

Reply:
it should re-start again